packages/lib-user/src/components/shared/ProjectCard/ProjectCard.js

```diff
@@ -0,0 +1,116 @@
+import { Box } from 'grommet'
+import { string } from 'prop-types'
+import styled from 'styled-components'
+import { SpacedText } from '@zooniverse/react-components'
+
+const StyledProjectCard = styled(Box)`
+  font-size: 12px;
+  height: 280px;
+  text-decoration: none;
+  width: 220px;
+
+  @media (768px < width < 1280px) {
+    font-size: 11.5px;
+    height: 240px;
+    width: 189px;
+  }
+
+  @media (width <= 768px) {
+    font-size: 11px;
+    height: 200px;
+    width: 157px;
+  }
+`
+
+const StyledProjectImage = styled(Box)`
+  height: 220px;
+
+  @media (768px < width < 1280px) {
+    height: 189px;
+  }
+
+  @media (width <= 768px) {
+    height: 157px;
+  }
+`
+
+const StyledProjectContent = styled(Box)`
+  position: static;
+  bottom: 0;
+  height: 22%;
+  transition: height 0.5s ease;
+
+  ${StyledProjectCard}:hover &,
+  ${StyledProjectCard}:focus & {
+    height: 65%;
+  }
+`
+
+const StyledSpacedText = styled(SpacedText)`
+  height: 0;
+  margin: 5px 0 0 0;
+  overflow: auto;
+
+  ${StyledProjectCard}:hover &,
+  ${StyledProjectCard}:focus & {
+    height: auto;
+  }
+`
+
+function ProjectCard ({
+  description = '',
+  displayName = '',
+  href = '',
+  imageSrc = ''
+}) {
+  return (
+    <StyledProjectCard
+      elevation='small'
+      forwardedAs='a'
+      href={href}
+      round='xxsmall'
+    >
+      <StyledProjectImage
+        background={{
+          image: `url(${imageSrc})`,
+          position: 'absolute',
+          size: 'cover',
+        }}
+        round={{ corner: 'top', size: 'xxsmall' }}
+      >
+      </StyledProjectImage>
+      <StyledProjectContent
+        flex='grow'
+        justify='center'
+        pad='xsmall'
+        round={{ corner: 'bottom', size: 'xxsmall' }}
+      >
+        <SpacedText
+          color={{ dark: 'neutral-6', light: 'dark-5' }}
+          size='inherit'
+          textAlign='center'
+          weight='bold'
+        >
+          {displayName}
+        </SpacedText>
+        <StyledSpacedText
+          color={{ dark: 'neutral-6', light: 'dark-5' }}
+          size='inherit'
+          textAlign='center'
+          uppercase={false}
+        >
+          {description}
```

Comment:
I'm not sure what to do about keyboard focus for the project description.
Currently, these changes do not include keyboard focus for the project description, which is relevant if the description overflows and a user would want to scroll to see the complete description.
Here's a video of the current changes in this PR:

https://github.com/zooniverse/front-end-monorepo/assets/12118751/cae8652d-162f-4419-ac95-3963b540765a

However, with some CSS and adding tabIndex to the description I can add keyboard focus, but then that element is always in the tab index order, and the content is announced twice with a screen reader (once when the card is focused, again when the text element is focused). Here's a video:

https://github.com/zooniverse/front-end-monorepo/assets/12118751/5ca401e4-3d28-4691-a6b5-41875d5d3cf1

I'm not sure which is preferable or if there's a different solution? The vast majority of project descriptions aren't that long, so this is related to an edge case.

Reply:
Discussed on Slack and concluded to use line-clamp to truncate project name at 3 lines and project description at 4 lines.